crates/foundry/evm/core/src/backend/mod.rs

```diff
@@ -696,6 +713,24 @@ impl Backend {
         false
     }
 
+    /// Whether the backend instance only executed pure cheatcodes.
+    /// Returns true if no cheatcodes were executed.
```

Comment:
```suggestion
    /// Returns true if no impure cheatcodes were executed.
```
?

Reply:
I meant to clarify with this comment that the function returns true in two cases:
- If cheatcodes were executed and only pure cheatcodes were executed.
- If no cheatcodes were executed.

Reply:
Ah, gotcha. Now it makes sense